### ISSUE TITLE
Warn if no source maps found

### ISSUE THREAD

**Issue by twschiller** [thredup/rollbar-sourcemap-webpack-plugin#39]
It would be helpful for the plugin to warn (or potentially fail) if no source maps are found. If the user includes the plugin in their Webpack configuration, it's reasonable to assume they intend for a source map to be uploaded

**Comment by brandondoran** [comment]
@twschiller Yes, I think that would be the reasonable way to handle the situations were no sourcemaps are found. I can add this in over the next week or so but, by all means, feel free to open up a PR as well.

**Comment by MerlinBB** [comment]
+1 for this - I managed to break my sourcemap uploading a while ago but I've only just noticed as it's been failing silently 😬
